app/src/main/res/layout/queue_fragment.xml

```diff
@@ -32,12 +32,20 @@
         android:layout_below="@id/info_bar"
         android:background="?android:attr/listDivider"/>
 
-    <de.danoeh.antennapod.view.EpisodeItemListRecyclerView
-        android:id="@+id/recyclerView"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:paddingHorizontal="@dimen/additional_horizontal_spacing"
-        android:layout_below="@id/divider" />
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+            android:id="@+id/swipeRefresh"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_below="@id/divider"
+            app:layout_behavior="@string/appbar_scrolling_view_behavior">
```

Comment:
`app:layout_behavior` is irrelevant on the queue screen, I think

Reply:
Yes, thanks. That should not have been there, will remove that, test again and commit.